regtests/mww3_test_01/input/ww3_shel.nml

```diff
@@ -22,7 +22,7 @@
 ! Define the output types point parameters via OUTPUT_TYPE_NML namelist
 ! -------------------------------------------------------------------- !
 &OUTPUT_TYPE_NML
-  TYPE%FIELD%LIST          = 'HS FP DP'
```

Comment:
@benoitp-cmc can you also add ICE to mww3_test_01/input/ww3_shel.inp?

Reply:
Done. You got me just in time, I'll be on vacation for a couple days, back to work on Thursday.

Reply:
Ps, @benoitp-cmc, I'm heading out on leave myself after today.  @JessicaMeixner-NOAA is graciously taking over the review of this PR for me.  Best wishes for your vacation. Cheers